### PR TITLE
Chore/handout corrections

### PR DESCRIPTION
**Pull request by igor-ka**
@winkerVSbecks  this is the handout portion of the changes.
If #20 is merged first this will be easier to look at.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +35,7 @@ You will then need to build the front end using:
 
 The above install npm modules, bower dependencies, and typings for the course
 
-Once you've done that, you can access the front-end of the project, by running a simple gulp task `gulp serve` within the root directory.
+Once you've done that, you can access the front-end of the project, by running a simple gulp task `gulp serve` within the *output* directory.
```
Comment (winkerVSbecks) [inline README.md]: You should be able to run this gulp task from the base directory? It's wired up to serve the `output` directory.

File: handout/00-setup.md [hunk 2]
```diff
@@ -123,7 +116,7 @@ Install `http-server` using `npm`:
 Fire up the server:
 
 ```sh
-  http-server client/
+  http-server output/dist
```
Comment (winkerVSbecks) [inline handout/00-setup.md]: Wouldn't `gulp serve` be serving the files already at this point?

File: handout/02-toolchain.md [hunk 3]
```diff
@@ -116,5 +110,5 @@ Finally, install `http-server` if you have not yet done so:
 We can run the app with:
 
 ```
-  http-server client/
+  http-server output/dist
```
Comment (winkerVSbecks) [inline handout/02-toolchain.md]: same question here RE: gulp serve vs http-server

File: README.md (hunk 1 above)
Comment (igor-ka) [inline README.md]: that is correct ill fix that

File: handout/00-setup.md (hunk 2 above)
Comment (igor-ka) [inline handout/00-setup.md]: this is the bundle directory, I will explain during the course. Gives them a chance to use http-server

File: handout/02-toolchain.md (hunk 3 above)
Comment (igor-ka) [inline handout/02-toolchain.md]: same as above

**Comment by winkerVSbecks** [comment]
:shipit:

**Pull request merged**